mkdocs.yml

```diff
@@ -103,7 +103,7 @@ nav:
   - 'Realtime Multiplayer': 'gameplay-multiplayer-realtime.md'
   - "Authoritative Multiplayer": "gameplay-multiplayer-server-multiplayer.md"
 #  - 'Turn-based Multiplayer': 'gameplay-multiplayer-turnbased.md'
-#  - 'In-app Purchase Validation': 'in-app-purchase-validation.md'
+  - 'In-app Purchase Validation': 'in-app-purchase-validation.md'
```

Comment:
I think we should make sure this is a top level menu item and label it just "IAP Validation".